src/test/kotlin/org/coepi/api/base/LocalDynamoDB.kt

```diff
@@ -0,0 +1,61 @@
+package org.coepi.api.base
+
+import com.amazonaws.AmazonServiceException
+import com.amazonaws.client.builder.AwsClientBuilder
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClientBuilder
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapper
+import com.amazonaws.services.dynamodbv2.document.DynamoDB
+import com.amazonaws.services.dynamodbv2.local.main.ServerRunner
+import com.amazonaws.services.dynamodbv2.model.*
+import org.coepi.api.dao.ReportsDaoTest
+
+class LocalDynamoDB {
+
+    fun startDynamoDb(){
+        val localUrl= "http://localhost:9000";
+        val region = "us-west-2"
+
+        println("Trying to start dynamo db at $localUrl")
+        System.setProperty("sqlite4java.library.path", "./build/libs")
+        val server = ServerRunner.createServerFromCommandLineArgs(arrayOf("-inMemory", "-port", "9000"))
+        server.start();
+
+        val ddb = AmazonDynamoDBClientBuilder.standard()
+                .withEndpointConfiguration(AwsClientBuilder.EndpointConfiguration(localUrl, region))
+                .build();
+        var ddbMapper = DynamoDBMapper(ddb);
+        createTable(ddb, ddbMapper)
+
+    }
+
+    /**
+     * TODO: Make it generic to create table based on class name passed to this method.
+     */
+    private fun createTable(ddb: AmazonDynamoDB, ddbMapper: DynamoDBMapper){
```

Comment:
Why do we need ReportsTableTest ? The idea is to unit test the table we use in the codebase.

Reply:
Report model will need some modifications for that. Some annotations like @DynamoDBTable(tableName="Report")
Also Hash key mapping for dynamo-mappe  to create it.